### PR TITLE
Load context from burp

### PR DESCRIPTION
**Pull request by Mindslave**
This PR is related to: https://github.com/zaproxy/zaproxy/issues/6430

This script allows the user to create a new context from a burp configuration file. 
Such files are commonly provided by programs on HackerOne to make sure that everyone stays in scope. 
With this script we can use those same files for ZAP.

### CODE REVIEW

**Comment by thc202** [comment]
The changelog should be updated.

**Comment by Mindslave** [comment]
I added the Changelog.md (which seems to be causing a conflict now for some reason) and squashed the commits.
I hope this is what you had in mind, I have never had to use the git rebase command before so I am a bit unsure if I did this right.

**Comment by kingthorin** [comment]
After your rebase the changelog should contain both lines:
```
- standalone/load_context_from_burp -> import context from burp config file
- Passive scan script for finding potential s3 Bucket URLs
```

**Comment by kingthorin** [comment]
Give it a shot, if you're still having trouble lemme know and I can tackle it for you.

**Comment by Mindslave** [comment]
Okay, so that did not have the desired effect, I was missing changes from the master branch (that's why I did not have the "- Passive scan script for finding potential s3 Bucket URLs") so I pulled and merged them, now It shows changes in 25 files, that shouldn't really be the case, should it?

**Comment by thc202** [comment]
Addressed. (For further changes you should first pull the latest changes from your remote fork.)

**Comment by Mindslave** [comment]
Thanks! Could you tell me what you did to fix it? I'm sorry for the inconvenience.

**Comment by thc202** [comment]
Just rebased on `master` and addressed the conflict, `git rebase upstream/master`. If you still have that locally you can try it out :)

**Comment by Mindslave** [comment]
Is there anything left, that needs to be done here?

**Comment by thc202** [comment]
Still needs to be reviewed.

File: standalone/load_context_from_burp.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+def get_host(include):
+	host = str(include['host'])
+	host = host[1:-1]
+	return host
+
+def get_protocol(include):
+	protocol = str(include['protocol'])
+	if protocol == "http":
+		protocol = "http:\/\/"
```
Comment (thc202) [inline standalone/load_context_from_burp.py:68]: `\/\/ `→ `//`, same for the other statement. It's easier to read (the slashes don't need to be escaped).

File: standalone/load_context_from_burp.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+def get_context_name():
+	ctx_name = JOptionPane.showInputDialog(None, "Enter a Name for the Context")
+	return ctx_name
+
+def get_file_name():
+	frame = JFrame("Filename")
+	frame.setLocation(100,100)
+	frame.setSize(500,400)
+	frame.setLayout(None)
```
Comment (thc202) [inline standalone/load_context_from_burp.py:22]: The frame is not really needed (you can pass `None` in the `showOpenDialog`.

**Comment by thc202** [comment]
This could be added to the new Export/Import add-on (in zap-extensions) instead (or in addition to?).

**Comment by Mindslave** [comment]
I added the changes you requested.
I'm not sure what the "new Export/Import add-on (in zap-extensions)" is, can you elaborate?

**Comment by thc202** [comment]
Thank you! The commits could be squashed (though can be done on merge too).

It's an add-on created recently, it's in https://github.com/zaproxy/zap-extensions/tree/03c816015082d97b23f457a779c301202ba0bb4f/addOns/exim which allows to import/export data (which this script is about). But adding the script here makes sense, if the user wants to do quick changes (or as base to adapt to other formats).

**Review by thc202 — APPROVED** [review]

**Review by psiinon — APPROVED** [review]

**Pull request merged**